src/ParcelRegistry.Projections.BackOffice/appsettings.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+  "ConnectionStrings": {
+    "Events": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;",
+    "BackOffice": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;",
+    "BackOfficeProjections": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;",
+    "BackOfficeProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;"
```

Comment:
should we use this opportunity to use only BackOffice here? Seeing as BackOffice, BackOfficeProjections and BackOfficeProjectionsAdmin are all pointing to the same connectionstring.

Reply:
BackOffice and BackOfficeProjections are different, I think they even go to different database.